### PR TITLE
Myahya story

### PR DESCRIPTION
**Pull request by myahyayaseen**
Please review this, @casey-collab

### CODE REVIEW

**Comment by myahyayaseen** [comment]
Please review this, @casey-collab !

File: urdu/cinderella-in-urdu.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This is my contribution to the story.
```
Comment (casey-collab) [inline urdu/cinderella-in-urdu.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline urdu/cinderella-in-urdu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by myahyayaseen** [comment]
Changes made. Please review this. @casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Comment by myahyayaseen** [comment]
Everything is good now. @casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Comment by myahyayaseen** [comment]
No. @casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Pull request closed without merging**